sklearn/linear_model/logistic.py

```diff
@@ -1331,7 +1370,7 @@ def fit(self, X, y):
             raise ValueError("Tolerance for stopping criteria must be "
                              "positive; got (tol=%r)" % self.tol)
 
-        X = check_array(X, accept_sparse='csr', dtype=np.float64)
+        X = check_array(X, accept_sparse='csr', dtype=np.float64, order='C')
```

Comment:
In `LogisticRegression`, it was already order to C for other solvers:

https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/linear_model/logistic.py#L1033